app/views/conditions/dashboard.erb

```diff
@@ -0,0 +1,98 @@
+<div class="container">
+  <h2>Weather Dashboard</h2>
+  <h4>Max Temperature</h4>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Max Temperature (F)</th>
+        <th>Average Number of Rides</th>
+        <th>Highest Number of Rides</th>
+        <th>Lowest Number of Rides</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% (40..80).step(10) do |n| %>
+      <tr>
+        <td><%= "#{n} to #{n + 10}" %></td>
+        <td><%= Condition.average_number_of_rides_from_temp(n) %></td>
```

Comment:
this is a lot of logic in your views - given that you're accessing a model. it's preferable to set this as an instance variable and pass it in via your controller. 

i know that given the way you have things up this may not be easily possible - but i encourage you to find another way of accessing this data